pipeline/data/parallel_importer.py

```diff
@@ -120,13 +106,47 @@ def add_alignments(corpus: List[str]) -> List[str]:
     return corpus_tsv
 
 
+def build_aug_mix(src: str, trg: str) -> Callable[[], CompositeModifier]:
+    """
+    Different types of scripts can use different types of mixed augmentations.
+    Determine which ones apply here, and create the appropriate mix.
+    """
+    src_script = get_script_info(src)
+    trg_script = get_script_info(trg)
+
+    assert src_script, "The script info must exist for the src language."
+    assert trg_script, "The script info must exist for the trg language."
+
+    logger.info("src_script " + repr(src_script))
+    logger.info("trg_script " + repr(trg_script))
+
+    modifiers: list[Modifier] = [
+        RemoveEndPunctuationModifier(MIX_PROB),
+        NoiseModifier(MIX_PROB),
```

Comment:
The order is important. Noise should go before the placeholder, near the end, otherwise we modify the added noise lines with other modifiers.